packages/smooth_app/pubspec.yaml

```diff
@@ -57,7 +57,11 @@ dependencies:
       path: "packages/camera/camera_platform_interface"
   scanner_shared:
     path: ../scanner/shared
-
+  app_store_shared:
+    path: ../app_store/shared
+  task_manager:
```

Comment:
This was in conflict, has it been added by this PR, or was it removed from main alltogether ?